components/meter/meter-linear.js

```diff
@@ -0,0 +1,53 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { meterStyles } from './meter-styles.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class MeterLinear extends RtlMixin(LitElement)  {
+	static get styles() {
+		return [ meterStyles ];
+	}
+
+	render() {
+		return html `
+			<svg class="loadingBar" viewBox="-5 -4.5 ${this.width} 9" width="${this.width}">
```

Comment:
As with Polymer, to be safe, we need to fully qualify classes in order to avoid potential targeting of this element by other styles potentially on the page in IE11 and Edge.

Reply:
Of course.